archetypes/business-service-spring-boot-starter/README.md

```diff
@@ -0,0 +1,18 @@
+## Welcome to use ServiceComb Java Chassis
+This project(module) is generate by *org.apache.servicecomb.archetypes:business-service-spring-boot-starter-archetype*, it use **spring-boot-starter-provider** to develop service endpoint.  
+
+### More works can be done further:
+1. Modify "HelloEndpoint", add your business service logic, or create some new endpoints to provide your services. More details can be found : http://servicecomb.incubator.apache.org/users/use-servicecomb-in-spring-boot/
```

Comment:
I think we make a lot of misunderstanding about integration with spring boot/spring cloud. Why, What, When, and How is not mentioned in docs. Here got some ServiceComb design concepts and spring boot/cloud integration concepts 
https://java.huaweicse.com/using-cse-in-spring-boot.html
https://bbs.huaweicloud.com/blogs/1fc9427c088611e89fc57ca23e93a89f
I am going to suggestion a way to integrate these docs into ServiceComb official documents

Reply:
@liubao68 Please feel free to polish the document if you have any other good idea about it.
You can write a simple for it to show a better way to work with Spring Boot/Cloud.